ui/package.json

```diff
@@ -95,7 +94,6 @@
     "ember-export-application-global": "^2.0.1",
     "ember-fetch": "^8.1.1",
     "ember-inflector": "^4.0.2",
-    "ember-inline-svg": "^1.0.1",
```

Comment:
Two packages we don't have to import anymore

Reply:
Nice!!